docs/en/operation/Server-Logging.md

```diff
@@ -197,6 +181,29 @@ You can see that the log files are put into different folders according to their
 in the folder `master`, worker logs are put in folder `worker`, etc. Within each folder, log files from
 different workers are distinguished by the IP/hostname of the machine on which the server has been running.
 
+#### Control Remote Logging Behavior with `log4j.properties`
```

Comment:
why four \# ? is this section intended to be a sub-section of the previous one `Verify the Log Server has Started`?

Reply:
Thanks I missed that.